hosts.yml

```diff
@@ -125,6 +125,16 @@
       path: /etc/my.cnf
       state: absent
 
+  - name: Copy SSL Certificates
```

Comment:
I think this can actually be removed. The packer AMI process should have copied these certs to the correct location. I probably added this ansible job a year or two ago because I didn't want to make a new AMI. If you launch an instance and connect to it before running the playbook, if the certs are there, then this can be removed.

Reply:
I checked and indeed certs are part of the ami. Will remove this